jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-helpers.c

```diff
@@ -40,11 +41,25 @@ ecma_typedarray_helper_dispatch_construct (const ecma_value_t *arguments_list_p,
 {
   JERRY_ASSERT (arguments_list_len == 0 || arguments_list_p != NULL);
   ecma_builtin_id_t proto_id = ecma_typedarray_helper_get_prototype_id (typedarray_id);
-  ecma_object_t *prototype_obj_p = ecma_builtin_get (proto_id);
+  ecma_object_t *current_new_target_p = JERRY_CONTEXT (current_new_target);
+  ecma_object_t *prototype_obj_p;
 
-  if (JERRY_CONTEXT (current_new_target))
+  if (current_new_target_p != NULL)
   {
-    prototype_obj_p = ecma_op_get_prototype_from_constructor (JERRY_CONTEXT (current_new_target), proto_id);
+    prototype_obj_p = ecma_op_get_prototype_from_constructor (current_new_target_p, proto_id);
+    if (jcontext_has_pending_exception ())
+    {
+      return ECMA_VALUE_ERROR;
+    }
+  }
+  else
+  {
+    prototype_obj_p = ecma_builtin_get (proto_id);
+  }
+
+  if (prototype_obj_p == NULL)
```

Comment:
This if is not needed, since `ecma_builtin_get (proto_id);` never returns NULL

Reply:
OK, then I nested into the if statement